addons/pandora/api.gd

```diff
@@ -8,7 +8,10 @@ const ScriptUtil = preload("res://addons/pandora/util/script_util.gd")
 
 signal data_loaded
 signal data_loaded_failure
-signal entity_added(entity: PandoraEntity)
+signal entity_added(entity:PandoraEntity)
+signal import_ended(response: String, imported_count: int)
+signal import_calculate_ended(response: Dictionary)
```

Comment:
rather than having `response`, we could have two separate signals instead: one for success, one for failure